pkgs/development/libraries/gtk/4.x.nix

```diff
@@ -80,7 +80,7 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "mirror://gnome/sources/gtk/${lib.versions.majorMinor version}/gtk-${version}.tar.xz";
-    sha256 = "umfGSY5Vmfko7a+54IoyCt+qUKsvDab8arIlL8LVdSA=";
+    sha256 = "sha256-KLNW1ZDuaO9ibi75ggst0hRBSEqaBCpaPwxA6d/E9Pg=";
   };
 
   patches = [
```

Comment:
The darwin patch does not apply. (Cherry-picked this onto nixos-unstable so it is actually feasible to build this.)

Reply:
Looks like it should be no longer necessary with https://gitlab.gnome.org/GNOME/gtk/-/commit/90183b35889961774178cc332169c18591d0542e